lib/ansible/modules/cloud/amazon/rds_snapshot.py

```diff
@@ -93,6 +127,35 @@
   returned: always
   type: str
   sample: us-west-2a
+availability_zones:
+  description: Provides the list of Availability Zones (AZs) where instances in the DB cluster snapshot can be restored.
+  returned: always
+  type: list
+  sample: [ 'us-east-2a', 'us-east-2b', 'us-east-2c' ]
+cluster_create_time:
+  description: Specifies the time when the DB cluster was created, in Universal Coordinated Time (UTC).
+  returned: always
+  type: str
+db_cluster_identifier:
+  description: Specifies the DB cluster identifier of the DB cluster that this DB cluster snapshot was created from.
+  returned: always
```

Comment:
It's not always returned though? Probably want `returned: when snapshot of a cluster is taken` or similar